.github/workflows/macOS-tests.yml

```diff
@@ -41,8 +41,10 @@ jobs:
           codespell  # See .codespellrc for args
       - name: SwiftLint
         run: |
-          # Adding --fix flag makes CI print only errors that cannot be fixed automatically
-          swiftlint --fix
+          # 1. Make all automated fixes that are possible
+          # 2. git diff to see what (if any) automated fixes were made
+          # 3. See https://github.com/realm/SwiftLint#xcode explains why the double run
+          swiftlint --fix --quiet && git diff && swiftlint --quiet
```

Comment:
Guys, are we sure that this is a good idea to silently change some code on PR? Does we have a sure that those fixes wouldn't broke a thing? Maybe it worth it to just fail the ci/cd if the swiftlint get stuck with its checks to made the contributor to fix it manually?

Reply:
We are NOT changing the code of the pull request because GitHub Actions do not write changes back into the pr.

We elected not to use pre-commit.ci would add fixed commits to the pull request.

Reply:
So why there's that `--fix quiet` flag then, just curious?

Reply:
If a new contributor (does not run pre-commit) creates a pull request on this project then:
1. We want their pr to fail if they break any enabled SwiftLint rule that this codebase already passes...
2. ___Unless___ SwiftLint can auto-fix their code because...
    1. We do not want to distract them from contributing over such a trivial fix and
    2. One of the project maintainers (who does run pre-commit ;-) ) will eventually auto-fix the issue.

Reply:
Yep, I got it now, thank you for elaboration